daemon/get-kubernetes-labels.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
I think you need a rebase, this is a file that was already merged.

Reply:
I already deleted the file. Look at the last commits.